### PR TITLE
Disable vote function in IBFT PoS

### PR DESCRIPTION
**Pull request by Kourin1996**
# Description

This PR disables vote function to invite/kick a validator in PoS

# Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [x] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Breaking changes

Disable `ibft propose` and `ibft candidates` commands

# Checklist

- [x] I have assigned this PR to myself
- [x] I have added at least 1 reviewer
- [x] I have tested this code
- [ ] I have updated the README and other relevant documents (guides...)
- [x] I have added sufficient documentation both in code, as well as in the READMEs

# Additional comments

Please post additional comments in this section if you have them, otherwise delete it

### CODE REVIEW

**Review by zivkovicmilos — APPROVED** [review]
Looks good 💯

File: consensus/ibft/snapshot.go [hunk 1]
```diff
@@ -191,87 +191,98 @@ func (i *Ibft) processHeaders(headers []*types.Header) error {
...
-			if snap.Set.Includes(h.Miner) {
-				continue
-			}
-		} else {
-			// we can only remove if they are part of the validators list
-			if !snap.Set.Includes(h.Miner) {
-				continue
+		if !snap.Equal(parentSnap) {
+			if err := saveSnap(h); err != nil {
+				return nil
```
Comment (Vuksan) [inline consensus/ibft/snapshot.go]: Isn't it strange that we return `nil` here if there is error, and not `err`?

**Review by Vuksan — APPROVED** [review]
LGTM

File: consensus/ibft/snapshot.go (hunk 1 above)
Comment (Kourin1996) [reply consensus/ibft/snapshot.go]: I agree with you. I fixed the code

**Pull request merged**